windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -143,10 +150,14 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 # unneeded docs that bloat the file size (and also seem to cause issues with unzipping).
 ###############################
 
-& $GEM_BAT update --system 2.4.8
-& $GEM_BAT install fluentd:0.14.1 --no-ri --no-rdoc --no-document
+# Install a specific version of serverengine with fluentd as there is a compatibility issue with later versions
+# See: https://github.com/fluent/fluentd/issues/1195
+& $GEM_BAT install serverengine:1.6.4 fluentd:0.14.1 --no-ri --no-rdoc --no-document
 & $GEM_BAT install windows-pr:1.2.5 win32-ipc:0.7.0 win32-event:0.6.3 win32-eventlog:0.6.6 win32-service:0.8.9 fluent-plugin-winevtlog:0.0.4 --no-ri --no-rdoc --no-document
-& $GEM_BAT install fluent-plugin-google-cloud:0.5.3 --no-ri --no-rdoc --no-document
+& $GEM_BAT install protobuf:3.6 google-protobuf:3.0 grpc:1.0.1 googleapis-common-protos:1.3.4 fluent-plugin-google-cloud:0.5.3 --no-ri --no-rdoc --no-document
+
+# See: https://github.com/google/protobuf/issues/2247
+cp $LIB_GCC_DLL $SD_LOGGING_AGENT_DIR_BIN
```

Comment:
This is a hack (sorry, workaround) for the above issue. Want to add a TODO to remove it once the issue is fixed?